system/nagios-status-check/README.md

```diff
@@ -0,0 +1,70 @@
+# SYSTEM Nagios status check SignalFx detectors
+
+## How to use this module
+
+```hcl
+module "signalfx-detectors-nagios-status-check" {
+  source      = "github.com/claranet/terraform-signalfx-detectors.git//system/nagios-status-check?ref={revision}"
+
+  environment = var.environment
+  notifications = var.notifications
+}
+
+```
+
+## Purpose
+
+Creates SignalFx detectors with the following checks:
+
+- Nagios check status
+
+This module has been designed to alert with the same behavior as [Nagios](https://nagios-plugins.org/doc/guidelines.html#AEN78), basically a gauge which is equal to 1 will trigger à `WARNING` and equal to 2 a `CRITICAL`. 
+
+On the agent side, you need to send your metrics with the [telegraf exec plugin][https://docs.signalfx.com/en/latest/integrations/agent/monitors/telegraf-exec.html] which includes a [parser for nagios](https://github.com/influxdata/telegraf/blob/master/plugins/parsers/nagios/parser.go). The metric is named `nagios_state.state`, you need to add an `extraDimensions` to your monitor in order to be able to differantiate multiple script states.
+
+Here is an example : 
+
+```yaml
+- type: telegraf/exec
+  intervalSeconds: 30
+  command: /usr/local/bin/scripts/check-ipmitool.pl
+  extraDimensions:
+    script: check_ipmitool
+  telegrafParser:
+    dataFormat: nagios
+```
+
+Ideally, use an `extraDimensions` which allows to easily identify the executed script.
```

Comment:
I think you can remove this line while you already say it just above : 

> you need to add an `extraDimensions` to your monitor in order to be able to differantiate multiple script states.

and I prefer "you need" than "ideally" while this is a requirement in my mind.

may be can you provide 1 config example with 2 examples instead of only one to show the diff between dimensions.